enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -4252,8 +4252,8 @@ class AdjointGenerator
     SmallVector<Value *, 8> pre_args = {0, 0, 0};
     std::vector<DIFFE_TYPE> argsInverted = {DIFFE_TYPE::CONSTANT,
                                             DIFFE_TYPE::CONSTANT};
-    std::vector<Instruction *> postCreate;
-    std::vector<Instruction *> userReplace;
+    SmallVector<Instruction *> postCreate;
```

Comment:
SmallVector requires a size in the type for lower llvm versions

Reply:
fixed